tests/test_models.py

```diff
@@ -1,270 +1,181 @@
+ON_GPU = True
 
-    # coverage setter check
-    model.set_preproc_func(lambda x: x - 1)  # do this for coverage
-    assert model.get_preproc_func()(1) == 0
-    # coverage setter check
-    model.set_preproc_func(None)  # do this for coverage
-    assert model.get_preproc_func()(1) == 1
 
-    predictor = CNNPatchPredictor(model=model, batch_size=1, verbose=False)
-    # don't run test on GPU
-    output = predictor.predict(
-        dataset,
-        return_probabilities=return_probabilities,
-        return_labels=return_labels,
-        on_gpu=on_gpu,
+def _get_temp_folder_path():
+    """Return unique temp folder path"""
+    new_dir = os.path.join(
+        rcParam["TIATOOLBOX_HOME"], f"test_model_patch_{int(time())}"
     )
-    predictions = output["predictions"]
-    if return_probabilities:
-        probabilities = output["probabilities"]
-        assert len(probabilities) == len(predictions)
-    if return_labels:
-        labels = output["labels"]
-        assert len(labels) == len(predictions)
+    return new_dir
 
 
 def test_create_backbone():
     """Test for creating backbone."""
     backbone_list = [
         "alexnet",
-        "resnet18",
-        "resnet34",
-        "resnet50",
-        "resnet101",
-        "resnext50_32x4d",
-        "resnext101_32x8d",
-        "wide_resnet50_2",
-        "wide_resnet101_2",
-        "densenet121",
-        "densenet161",
-        "densenet169",
-        "densenet201",
-        "googlenet",
-        "mobilenet_v2",
-        "mobilenet_v3_large",
-        "mobilenet_v3_small",
+        # "resnet18",
+        # "resnet34",
+        # "resnet50",
+        # "resnet101",
+        # "resnext50_32x4d",
+        # "resnext101_32x8d",
+        # "wide_resnet50_2",
+        # "wide_resnet101_2",
+        # "densenet121",
+        # "densenet161",
+        # "densenet169",
+        # "densenet201",
+        # "googlenet",
+        # "mobilenet_v2",
+        # "mobilenet_v3_large",
+        # "mobilenet_v3_small",
     ]
     for backbone in backbone_list:
         try:
             get_model(backbone, pretrained=False)
         except ValueError:
-            assert False, "Model %s failed." % backbone
+            assert False, f"Model {backbone} failed."
 
     # test for model not defined
     with pytest.raises(ValueError, match=r".*not supported.*"):
-        get_model("secret_model", pretrained=False)
+        get_model("secret_model-kather100k", pretrained=False)
 
 
-def test_predictor_crash():
-    """Test for crash when making predictor."""
-    # test abc
-    with pytest.raises(NotImplementedError):
-        ModelBase()
-    with pytest.raises(NotImplementedError):
-        ModelBase.infer_batch(1, 2, 3)
-
-    # without providing any model
-    with pytest.raises(ValueError, match=r"Must provide.*"):
-        CNNPatchPredictor()
+def test_DatasetInfo():
+    """Test for kather patch dataset."""
+    # test defining a subclass of dataset info but not defining
```

Comment:
I'm actually no longer sure whether it makes sense to add this Kather patch dataset now - this is a different dataset to the kather 100k that we used. Let's have it in this PR but reconsider later.